api-server/src/roster/models.py

```diff
@@ -1,8 +1,10 @@
 from django.db import models
-from django.core.validators import MinValueValidator, MaxValueValidator
+from django.core.validators import MinValueValidator, MaxValueValidator, BaseValidator
 from django.contrib.auth.models import User
 from datetime import datetime, date
-
+from django.db.models.fields import TextField
```

Comment:
```suggestion
```

not required if the comment below to use `models.TextField` is accepted

Reply:
👍